src/routes/tools/degree-days/_ExploreData.svelte

```diff
@@ -141,6 +139,12 @@
 
   $: formatFn = format(`.${$climvar.decimals}f`);
 
+  $: indicatorTitle = $indicator.title.replace("Degree Days ", "");
```

Comment:
The chart title needs to indicate Heating Degree Days or Cooling Degree Days.

Reply:
This is for the MadLib next to the locator map, the chart title is created in the `yAxis.label` prop.

Reply:
MadLib 😄 . That's a perfect description. The Heating/Cooling needs to be in the madlib also.

Reply:
Yeah, that's how I think of these types of dynamic sentences :)

It's there, I'm just removing an extra "Degree Days" from the title:

<img width="810" alt="Screen Shot 2021-09-17 at 4 11 47 PM" src="https://user-images.githubusercontent.com/161748/133862629-375cc9c3-e406-4a8d-ad12-6ee45d93b456.png">

<img width="816" alt="Screen Shot 2021-09-17 at 4 11 56 PM" src="https://user-images.githubusercontent.com/161748/133862635-8275c907-ab25-4650-955b-10fbddbf0aec.png">

Reply:
<img width="800" alt="image" src="https://user-images.githubusercontent.com/161748/133862661-312f019a-483d-4832-8a2a-9a96b1ca8a02.png">